recipes-kernel/linux/linux-nilrt.inc

```diff
@@ -47,14 +47,14 @@ SRCREV ?= "${AUTOREV}"
 PV = "${LINUX_VERSION}+git${SRCPV}"
 
 LIC_FILES_CHKSUM = "file://COPYING;md5=6bc538ed5bd9a7fc9398086aedcd7e46"
-LIC_FILES_CHKSUM_xilinx-zynq = "file://COPYING;md5=d7810fab7487fb0aad327b76f1be7cd7"
+LIC_FILES_CHKSUM:xilinx-zynq = "file://COPYING;md5=d7810fab7487fb0aad327b76f1be7cd7"
 
 # This checks ${PV} == version from kernel sources which our PV/AUTOREV breaks, so skip it.
 KERNEL_VERSION_SANITY_SKIP="1"
 
-FILESEXTRAPATHS_prepend := "${THISDIR}/${KBUILD_FRAGMENTS_LOCATION}:"
-KBUILD_DEFCONFIG_x64 = "nati_x86_64_defconfig"
-KBUILD_DEFCONFIG_armv7a = "nati_zynq_defconfig"
+FILESEXTRAPATHS:prepend := "${THISDIR}/${KBUILD_FRAGMENTS_LOCATION}:"
+KBUILD_DEFCONFIG:x64 = "nati:x86_64_defconfig"
```

Comment:
Should the value have not been replaced?

Reply:
I knew I was going to miss one. :(

Will fix.